lib/kibana/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "detection-rules-kibana"
-version = "0.4.1"
+version = "0.4.2"
```

Comment:
With the latest changes I think these needs to changed

Reply:
This was inferred as a detection_rules pyproject.toml file during first pass. 
The kibana pyporject.toml can be a patch bump